README.md

```diff
@@ -9,11 +9,34 @@ It is used heavily by k6 extension developers as well as anyone who wishes to ma
 
 Stay updated, be aware of changes, and please submit feedback! Thanks!
 
-## Requirements
+## Docker
+
+The easiest way to use xk6 is via our [Docker image](https://hub.docker.com/r/grafana/xk6/). This avoids having to setup a local Go environment, and install xk6 manually.
+
+For example, to build a k6 v0.43.0 binary with the [xk6-kafka](https://github.com/mostafa/xk6-kafka) and [xk6-output-influxdb](https://github.com/grafana/xk6-output-influxdb) extensions, you would run:
+
+```bash
+docker run --rm -it -u "$(id -u):$(id -g)" -v "$PWD:/xk6" grafana/xk6 build v0.43.0 \
```

Comment:
Non-blocking suggestion

```suggestion
docker run --rm -it -u "$(id -u):$(id -g)" -v "${PWD}:/xk6" grafana/xk6 build v0.43.0 \
```

If I got [this correctly](https://stackoverflow.com/a/41489151) (I don't have windows with docker installed), it should work both on linux/windows power shell

Reply:
This _might_ work on PowerShell, but not on cmd.exe, which uses `%cd%` instead of `$PWD`. I'm actually surprised `id` exists on Windows, so those commands should actually work. :crossed_fingers: